Probesys/bileto
# new: Provide a dark mode

Changes proposed in this pull request:

- Fix the `@` alias in Vite configuration
- Increase line-height of .anchor-action
- Create a Twig extension to provide a `icon()` function (it just copies SVG files in the HTML for now)
- Provide a popup mechanism and use it for the logout button in the header
- Add a `colorScheme` attribute to users and a "preferences" form to change it
- Use this color scheme to adapt the colors of the application (light or dark)

How to test the feature manually:

1. Login to the app and go to the preferences page
2. Check that color scheme is set to `auto`
3. According to your OS preferences, check the theme is light or dark
4. Change your OS color scheme preferences and check the colors change in the application as well
5. Change the color scheme in the app, and check that the colors aren't affected by your OS preferences

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
